crates/oxc_linter/src/snapshots/no_render_return_value.snap

```diff
@@ -0,0 +1,58 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_render_return_value
+---
+  ⚠ eslint-plugin-react(no-render-return-value): Do not depend on the return value from ReactDOM.render.
+   ╭─[no_render_return_value.tsx:1:1]
+ 1 │ var Hello = ReactDOM.render(<div />, document.body);
+   ·     ───────────────────────
```

Comment:
The spans are incorrect 😅 , it should point it `ReactDOM.render`.

Reply:
Ah, I got it now.

Is it possible to add a function like `merge` to `Span`?

```rust
    pub fn merge_min_end(&mut self, other: &Self) -> Self {
        Self::new(self.start.min(other.start), self.end.min(other.end))
    }

    pub fn merge(&mut self, other: &Self) -> Self {
        Self::new(self.start.min(other.start), self.end.max(other.end))
    }
```

This will be useful in composing different spans.